corona-warn-app/cwa-app-android
# Overlapping infobox (EXPOSUREAPP-9628)

This PR addresses Exposureapp-9628.
(Overlapping Infobox above the qr codes)

How to test:
- Scan a certificate (QR)
- Press the Certificate tab in the bottom bar
- Check if the infobox above the qr code matches

![image](https://user-images.githubusercontent.com/46747780/134872817-b1cd5026-5354-44f9-ae8d-66a01015c5da.png)
